theano/tensor/nnet/nnet.py

```diff
@@ -2230,31 +2229,23 @@ class of the input x while the second predicts the output of the input x in
 
     The outputs are grouped in the same order as they are initially defined.
 
-    .. versionadded:: 0.7.1
+    .. versionadded:: 0.7.1 
```

Comment:
Don't add space at the end of lines. This break the code format we use and is probably what make travis test fail. Here is the failure in travis:

/home/travis/build/Theano/Theano/theano/tensor/nnet/nnet.py:2232:28: W291 trailing whitespace
/home/travis/build/Theano/Theano/theano/tensor/nnet/nnet.py:2286:21: E201 whitespace after '('
/home/travis/build/Theano/Theano/theano/tensor/nnet/nnet.py:2286:40: E202 whitespace before ')'
/home/travis/build/Theano/Theano/theano/tensor/nnet/nnet.py:2287:31: E201 whitespace after '('
/home/travis/build/Theano/Theano/theano/tensor/nnet/nnet.py:2287:50: E202 whitespace before ')'